pkg/controller/deployment/deployment_controller.go

```diff
@@ -124,6 +125,25 @@ func (r *ReconcileDeployment) Reconcile(request reconcile.Request) (reconcile.Re
 
 		jaeger := inject.Select(dep, ns, jaegers)
 		if jaeger != nil && jaeger.GetDeletionTimestamp() == nil {
+			if jaeger.Namespace != request.Namespace {
+				log.WithFields(log.Fields{
+					"jaeger namespace": jaeger.Namespace,
+					"app namespace":    request.Namespace,
+				}).Info("different namespaces, so check whether trusted CA bundle configmap should be created")
+				if cm := ca.GetTrustedCABundle(jaeger); cm != nil {
+					// Update the namespace to be the same as the Deployment being injected
+					cm.Namespace = request.Namespace
+					jaeger.Logger().WithFields(log.Fields{
+						"configMap": cm.Name,
+						"namespace": cm.Namespace,
+					}).Debug("creating Trusted CA bundle config maps")
+					if err := r.client.Create(ctx, cm); err != nil && !errors.IsAlreadyExists(err) {
+						log.WithField("namespace", request.Namespace).WithError(err).Error("failed to create trusted CA bundle")
+						return reconcile.Result{}, tracing.HandleError(err, span)
```

Comment:
Should it really skip the remaining of the reconciliation logic when the CA bundle config map fails to get created?

Reply:
Have removed - but unclear the impact this would have on app? If a problem we can also add back in.

Reply:
Not sure as well. Would the sidecar prevent the pod from reaching a stable state, if it can't get the bundle? @rubenvp8510 might have a better insight.

Reply:
I don't think it prevents pod to reach an stable state,  If I recall correctly this will never inject the side car but will retry the reconciliation process again and again.  This code is previous to update or inject the sidecar, and the` Inject.Select` doesn't have any side effect on the deployment.

Reply:
@rubenvp8510 unclear whether you prefer this approach, aborting the reconciliation or allowing the reconciliation to continue? (as it does now after the latest commit)?

Reply:
Sorry if I was unclear, I would prefer the approach of aborting the reconciliation. I think it is better to abort the process than having a sidecar that don't work due an unexisting configMap.

Reply:
No problem - just wanted to be sure before updating :) Thanks.